### ISSUE TITLE
[BUG] Hanime extension : episodes are seen as whole series

### ISSUE THREAD

**Issue by castrik** [jmir1/aniyomi#75]
I opened an issue here since its related to an extension not officially supported by tachiyomi
 
## Device information
- Tachiyomi version: all versions
- Android version: 11

## Source information
- Name: Hanime
- Extension version: 11.6

## Steps to reproduce
1. just open it

### Expected behavior
Describe what should have happened

series should be shown individually and all episodes should be in a series, its ok if it is divided by different seasons

### Actual behavior
Describe what happens instead

all episodes are shown as individual series and when we go in episode list, it just shows that single episodes

**Comment by ty-coon** [comment]
Yes and it makes the single episode a whole anime and then values in tracker overlaps.

**Comment by jmir1** [comment]
not a bug, that's just how the website works
